go.mod

```diff
@@ -3,22 +3,28 @@ module github.com/Mellanox/nic-configuration-operator
 go 1.21
 
 require (
+	github.com/jaypipes/ghw v0.12.0
+	github.com/k8snetworkplumbingwg/sriov-network-device-plugin v3.0.0+incompatible
```

Comment:
should we just shamelessly copy the dependencies ? we dont claim to have stable API for pkgs in device plugin.

Reply:
Copied the function locally